scripts/10-stunnel-increase-number-of-file-descriptors.conf

```diff
@@ -1,2 +1,6 @@
+[Unit]
+Requires=gencert.service
```

Comment:
This can be put into gencert.service itself if you invert the direction, i.e. instead of Requires say RequiredBy.
Ask @TimSmithCtx or @rosslagerwall  for the preferred way to do this with systemd.

Reply:
yes, it looks like i want `RequiredBy`

Reply:
inverting the dependencies means that we have to enable gencert for the following symlink to be created:

```
/etc/systemd/system/stunnel@xapi.service.requires/gencert.service to /usr/lib/systemd/system/gencert.service.
```

if we don't enable it, we get an error in daemon.log because the cert never gets created.

whereas if we just leave the gencert dependency in the stunnel@xapi drop-in, we don't have to worry about enabling gencert (so i would suggest that this is preferred)